### PR TITLE
Updated AutoCompleteSelectMultipleWidget.render to work well with strings

### PR DESCRIPTION
**Pull request by u8sand**
References #242

### CODE REVIEW

File: ajax_select/fields.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def render(self, name, value, attrs=None, renderer=None, **_kwargs):
 
         lookup = registry.get(self.channel)
 
-        values = list(value)
+        values = value.split(',') if type(value) == str else list(value)
```
Comment (crucialfelix) [inline ajax_select/fields.py]: It actually isn't the job of the widget to parse the GET request. This should not happen here, the value must be a list or an iterable.

But anyway you would have to do this:
```
from django.utils import six
# ...
... if isinstance(value, six.string_types) ...
```

You would also need to collect the split values and safely parse as int.

Comment (crucialfelix) [reply ajax_select/fields.py]: You say you have a URL with comma separated values, but nothing in Ajax Selects generates that URL. 

The internal representation used in the form field is | delimited:

https://github.com/crucialfelix/django-ajax-selects/blob/develop/ajax_select/fields.py#L250

I think what you want is to use the widget's `value_from_datadict`

I'm pretty sure that the form field already uses the dict supplied to the form and calls this on the widget to get the value.

ie. I think you want to use | not , to separate values.

Comment (u8sand) [reply ajax_select/fields.py]: I'll look into this soon. I'm fine with re-using the `value_from_datadict` function for this purpose and investigate why it wasn't interpreted as a list by the time it made it to `ajax_select`.

Comment (crucialfelix) [reply ajax_select/fields.py]: Any parsing of the URL or GET should have been done before it reaches this point.

**Comment by u8sand** [comment]
So I'm instantiating a `ModelForm` from GET parameters to enable the form to get pre-populated.

```python
form = form_cls(initial=dict(self.request.GET.dict(), **{
  'authors': [self.request.user],
}))
```

Turns out something depends on `initial` being a `QueryDict`, not a normal `dict`.

Changing this to
```python
GET = self.request.GET.copy()
GET['authors'] = self.request.user
form = form_cls(initial=GET)
```

Made it work with the existing django-ajax-selects.

**Pull request closed without merging**